### PR TITLE
Arktrade Bug

### LINKED ISSUE #998: [Bug]
**Describe the bug**
Bug on arktrades.

**To Reproduce**

This command seems to crash :
```bash
stocks > dd > arktrades
```

**Screenshots**
Results :
```bash
(✨) (stocks)>(dd)> arktrades
'str' object has no attribute 'strftime' 
```

**Desktop (please complete the following information):**
- OS: windows
- Python version = 3.8

**Additional context**
```python
from gamestonk_terminal.stocks.due_diligence import ark_model

df = ark_model.get_ark_trades_by_ticker(ticker="TSLA")
```

`Closed` data seems to be fetched properly with `yf.download` function.

`Total` column is calculated with `Closed` column.

They both are set to `NaN` in the returned `DataFrame`.

### PR DESCRIPTION
**Pull request by jmaslek**
This fixes #998.  Issue was that the indexes were different date time objects.  Beats me why it randomly stopped working.

Also fixed an error where the tsxv package is no longer imported, so the prediction crashes.

### CODE REVIEW

**Review by Chavithra — APPROVED** [review]
Arktrade fix worked on my side.

File: gamestonk_terminal/common/prediction_techniques/regression_model.py [hunk 1]
```diff
@@ -2,11 +2,60 @@
 __docformat__ = "numpy"
 from typing import Tuple, List, Any, Union
 import pandas as pd
-from tsxv import splitTrain
+import numpy as np
 from sklearn import linear_model
 from sklearn import pipeline
 from sklearn import preprocessing
 
+# The tsxv dependence was removed so this fails.  Taking from didiers source
```
Comment (DidierRLopes) [inline gamestonk_terminal/common/prediction_techniques/regression_model.py:10]: I'm gonna sue you 

image

File: gamestonk_terminal/common/prediction_techniques/regression_model.py [hunk 2]
```diff
@@ -2,11 +2,60 @@
...
+
+
+def split_train(
+    sequence: np.ndarray, numInputs: int, numOutputs: int, numJumps: int
+) -> Tuple[List, List]:
+    """Returns sets to train a model
```
Comment (DidierRLopes) [inline gamestonk_terminal/common/prediction_techniques/regression_model.py:16]: Using our numpy docstring perhaps?

File: gamestonk_terminal/common/prediction_techniques/regression_model.py (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/common/prediction_techniques/regression_model.py:10]: > 

do It u won't

**Pull request merged**